include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -1078,22 +1068,6 @@ void RunInParallel(std::function<void(unsigned idx)> fn, unsigned n, std::ptrdif
   profiler_.LogEnd(ThreadPoolProfiler::WAIT);
 }
 
-void Cancel() override {
-  cancelled_ = true;
```

Comment:
Would we need this if we did anything more for parallel and/or async execution?

Reply:
I think it is mainly needed for fast failing.  Suppose you have summited a "parallel for" code block with 4 threadpool  tasks to a thread pool, and one of the tasks failed. If you want to quickly cancel the other 3 without continue to wait them being finished, then a cancel method would help. If everything was fine, then we don't need to cancel the pending or running tasks.